data/mods/Xedra_Evolved/items/ammo.json

```
@@ -0,0 +1,15 @@
+[
+  {
+    "id": "ss_57mm",
+    "copy-from": "57mm",
+    "type": "AMMO",
+    "name": { "str_sp": "5.7x28mm SB193" },
+    "description": "A subsonic cartridge derived from the 5.7x28mm, designed for silenced firearms.  It offers good armor penetration at the cost of slightly less damage.",
+    "relative": {
+      "price": 100,
+      "range": -1,
+      "damage": { "damage_type": "bullet", "amount": -3 },
```

```suggestion
      "damage": { "damage_type": "bullet", "amount": -10 },
```
https://web.archive.org/web/20021212233833/http://www.fnmfg.com/lawenf/ss190/ss190.htm
.5 * 0.0036kg x (300m/s)^2 = 162J(12.7) vs .5*.00202kg *(715m/s)^2 = 516J(22.7)

Note: ammo it copies from is currently 20 when it should be that 22, ewps 🤪